src/EnergyPlus/Plant/Component.cc

```diff
@@ -121,5 +121,10 @@ namespace DataPlant {
         }
     }
 
+    CompData &CompData::getPlantComponent(EnergyPlusData &state, PlantLocation const &plantLoc)
+    {
+        return state.dataPlnt->PlantLoop(plantLoc.loopNum).LoopSide(plantLoc.loopSideNum).Branch(plantLoc.branchNum).Comp(plantLoc.compNum);
+    }
```

Comment:
Implemented a `getPlantComponent()` function to shorten the `state.dataPlnt->PlantLoop(plantLoc.loopNum).LoopSide(plantLoc.loopSideNum).Branch(plantLoc.branchNum).Comp(plantLoc.compNum)` calls...